x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/source_logic.test.ts

```diff
@@ -65,28 +84,43 @@ describe('SourceLogic', () => {
     it('setContentSource', () => {
       SourceLogic.actions.setContentSource(contentSource);
 
-      expect(SourceLogic.values.contentSource).toEqual(contentSource);
-      expect(SourceLogic.values.dataLoading).toEqual(false);
+      expect(SourceLogic.values).toEqual({
+        ...defaultValues,
+        contentSource,
+        dataLoading: false,
+      });
     });
 
     it('onUpdateSourceName', () => {
       const NAME = 'foo';
       SourceLogic.actions.setContentSource(contentSource);
       SourceLogic.actions.onUpdateSourceName(NAME);
+      const EXPECTED_STATE_BASE = setExpectedState({ setContentSource: true });
```

Comment:
I question whether this should be a constant here. Any objection to changing these to `const expectedStateBase = ...`?